### PR TITLE
Update the Risk of dangerous-workflow

### PR DESCRIPTION
**Pull request by dota17**
1.Update the Risk of dangerous-workflow 

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
#1348  

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: docs/checks.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ participants.
```
Comment (azeemshaikh38) [inline docs/checks.md:261]: This is an is auto-generated file and shouldn't be modified directly.  You will need to change `docs/checks/internal/checks.yaml` and run `make generate-docs` instead.

File: docs/checks.md [hunk 2]
```diff
@@ -528,7 +528,7 @@ information about a bug is not publicly visible.
 **Remediation steps**
 - Place a security policy file `SECURITY.md` in the root directory of your repository. This makes it easily discoverable by a vulnerability reporter.
 - The file should contain information on what constitutes a vulnerability and a way to report it securely (e.g. issue tracker with private issue support, encrypted email with a published public key).
-- For GitHub, see more information [here](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository).
+- For GitHub, see more information [here](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository.).
```
Comment (laurentsimon) [inline docs/checks.md:531]: can you make the change in checks.yml and re-generate the docs?

**Review by laurentsimon — COMMENTED** [review]
Thanks for catching the out-of-sync with the docs!
Revert of your .md changes, make them in the yml like @azeemshaikh38 suggested, and we'll be ready to merge.
Thanks for you help!
If you're interested in contributing to other issues, let us know; we'll be happy to suggest.

**Review by azeemshaikh38 — APPROVED** [review]
Thanks!

**Pull request merged**